.github/workflows/test-seahorn-docker.yml

```diff
@@ -0,0 +1,55 @@
+# workflow to build and test seahorn
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: dev10
+  pull_request:
+    branches: dev10
+  schedule:
+    - cron: 0 0 * * *  # run every day at UTC 00:00
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "test"
+  test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Check Out Repo 
+        uses: actions/checkout@v2
+        with:
+          ref: dev10  # only checkout dev10
```

Comment:
will this work on PRs? Maybe we should have separate actions. One for PR, one cron. The cron one can use a fixed branch.

Reply:
oh, this only lives in master